Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccreissuance/ui/success/DccReissuanceSuccessFragment.kt

```diff
@@ -1,6 +1,25 @@
 package de.rki.coronawarnapp.dccreissuance.ui.success
 
+import android.os.Bundle
+import android.view.View
 import androidx.fragment.app.Fragment
 import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.FragmentDccReissuanceSuccessBinding
+import de.rki.coronawarnapp.util.ui.popBackStack
+import de.rki.coronawarnapp.util.ui.viewBinding
 
-class DccReissuanceSuccessFragment : Fragment(R.layout.fragment_dcc_reissuance_success)
+class DccReissuanceSuccessFragment : Fragment(R.layout.fragment_dcc_reissuance_success) {
```

Comment:
I can do that, yes. However, there is so little content on this screen so that even on small devices and big font sizes the user wouldn't need to scroll to see everything. What do you think?

Reply:
ok